### DISCUSSION TITLE
Port to MacOS

### DISCUSSION THREAD

**Discussion by The-OMG** [leo-arch/clifm#30] [Ideas]
Hello, this is a pretty awesome project. Would it be possible to make it available in the brew package manager.

**Comment by leo-arch** [comment]
Hi @The-OMG. That would be great, and it's definitely in our plans. But we need to find first a package maintainer and tester for `MacOS`. Indeed, I'm not even sure this thing will work out of the box on `MacOS`. Perhaps it will, but I'm not sure: just didn't tested it. If you happen to know someone able to compile the program on `MacOS` and build a package for `brew`, that's would be a great contribution.

**Comment by leo-arch** [comment]
Hi @The-OMG! If you're still interested, **clifm** is available in [hombrew](https://formulae.brew.sh/formula/clifm) since version `1.10`.
